### PR TITLE
Demo image files never picked up

### PR DESCRIPTION
**Pull request by olavea**
I tested in gatbsy develop that the book is building pages from images in "demo" not in "theme"

### CODE REVIEW

**Comment by raae** [comment]
Great job on #8.

The SEO component should be used with a title prop. In this case, it should be the filename. The `title` props is a little misleading as it refers to the page title, not the site title. The Helmet component then combines the page title with the site title using the `titleTemplate`.

If you send in the filename as the `title` prop the title will end up being "1 | gatsby-theme-childrens-picture-book".

File: theme/gatsby-config.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
 module.exports = {
   siteMetadata: {
-    title: "Gatsby Theme Jam Example Submission",
+    title: "gatsby-theme-childrens-picture-book",
+    description:
+      "Make a Do-It-Yourself childrens picture book together with a child you know.",
```
Comment (raae) [inline theme/gatsby-config.js]: Spelling error "children's" not "childrens".

File: theme/src/templates/page.js [hunk 2]
```diff
@@ -2,13 +2,15 @@ import React, { Component } from "react"
...
 import PageFlipper from "../components/PageFlipper"
+import SEO from "../components/seo"
 
 export default class postLayout extends Component {
   render() {
     const { file } = this.props.data
 
     return (
       <>
+        <SEO />
```
Comment (raae) [inline theme/src/templates/page.js]: There should be a title prop added here set to equal `file.name`.

File: theme/gatsby-config.js (hunk 1 above)
Comment (raae) [inline theme/gatsby-config.js:5]: Spelling error "children's" not "childrens".

**Review by raae — CHANGES_REQUESTED** [review]

File: theme/src/templates/page.js (hunk 2 above)
Comment (olavea) [reply theme/src/templates/page.js]: page.js
`<title titleProp={file.name} />`

File: theme/gatsby-config.js (hunk 1 above)
Comment (olavea) [reply theme/gatsby-config.js:5]: theme/gatsby-config.js
`...
siteMetadata: {
    title: "gatsby-theme-children's-picture-book",
...`

**Review by raae — APPROVED** [review]

**Pull request merged**